File: FaceSample/app/src/main/java/com/regula/facesample/items/basic/FaceCaptureDefaultItem.java
package com.regula.facesample.items.basic;

import android.content.Context;

import com.regula.facesample.items.CategoryItem;
import com.regula.facesdk.FaceSDK;

/**
 * Created by Sergey Yakimchik on 20.08.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class FaceCaptureDefaultItem extends CategoryItem {

    @Override
    public void onItemSelected(Context context) {
        FaceSDK.Instance().presentFaceCaptureActivity(context, faceCaptureResponse -> { });
    }

    @Override
    public String getTitle() {
        return "FaceCapture";
    }

    @Override
    public String getDescription() {
        return "Automaticlly captures face photo.";
    }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/items/basic/FaceCaptureDefaultItem.kt
package com.regula.facesamplekotlin.items.basic

import android.content.Context
import com.regula.facesamplekotlin.category.CategoryItem
import com.regula.facesamplekotlin.util.FaceCaptureResponseUtil
import com.regula.facesdk.FaceSDK

class FaceCaptureDefaultItem : CategoryItem() {

    override fun onItemSelected(context: Context) {
        FaceSDK.Instance().presentFaceCaptureActivity(context) { response ->
            // ... check response.image for capture result
            FaceCaptureResponseUtil.response(context, response)
        }
    }

    override val title: String
        get() = "FaceCapture"

    override val description: String
        get() = "Automatically captures face photo."

}